### PR TITLE
Add ttbar and ZG skims

### PR DESCRIPTION
**Pull request by Andrew42**
This PR adds skims for the newly added ttbar and ZG samples. It also updates xsec values in the original un-skimmed JSONs as well as in the `xsec.cfg` file. Finally, it adds a couple of utility functions to `utils.py` that I've wanted to get merged for a while now, but shouldn't really impact anything else in the repo.

The new cross-sections are:
TTTo2L2Nu: 87.31483776
TTToSemiLeptonic: 364.35080448
ZGToLLG: 55.78

**Note:** I wasn't able to confirm the source for the ZG number, so @sscruz could you please confirm this is the correct value and (if possible) where it came from?

The python command used to update the json files:
```
python make_skim_jsons.py --file ND_bkgd_mc_skim_locations.txt --json-dir ../../topcoffea/json/background_samples/central_UL
```

### CODE REVIEW

**Review by bryates — APPROVED** [review]
Looks good, thanks for adding these.

**Comment by Andrew42** [comment]
Also, if we want to include an actual link to the TOP-17-001 paper (for the ttbar incl. xsec) in the `xsec.cfg`, it should be https://arxiv.org/abs/1812.10505

**Comment by kmohrman** [comment]
This looks good to me but I've realized that for the unskimmed samples, I forgot to run them through the sum of the weights processor to get the sum of nominal and up/down weights. I can do that on this branch (or a different branch) and update the unskimmed JSONs, but I'm not sure of the best way to update the skimmed JSONs? @Andrew42

**Comment by Andrew42** [comment]
if you update the weights on this branch, I can just re-run the skim json maker and it should update everything automatically

**Comment by kmohrman** [comment]
Sounds good, I'll check out the branch and get the jobs running now. Also, regarding your above comment about the ZG xsec, I'd be ok with merging this PR even if we can't track down the reference (and just leaving the issue about it open). I think we want to see how this affects the CRs ASAP, and we don't want to be held up while tracking down the xsec reference.

**Comment by Andrew42** [comment]
That's fine with me if you want to merge it before tracking down a reference. I just wanted to make sure it was mentioned so that we have some sort of record for it. I'm sure you can sympathize with the struggles of tracking down cross section references.

The best time to start documenting things was yesterday, the 2nd best time is right now!

File: topcoffea/cfg/mc_background_samples.cfg [hunk 1]
```diff
@@ -3,83 +3,95 @@
...
+#../../topcoffea/json/background_samples/central_UL/UL16APV_TTJets.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_tW.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WJetsToLNu.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WWTo2L2Nu.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WWW_4F.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WWZ_4F.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WZTo3LNu.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_WZZ.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_ZZTo4L.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_ZZZ.json
```
Comment (bryates) [inline topcoffea/cfg/mc_background_samples.cfg:25]: Looks like you pushed this by accident. Can you uncomment everything?

Comment (kmohrman) [reply topcoffea/cfg/mc_background_samples.cfg]: Yep, sorry, already done

Comment (bryates) [reply topcoffea/cfg/mc_background_samples.cfg]: Thanks!

**Comment by kmohrman** [comment]
Thanks @Andrew42. Could you add the skimmed jsons to the skimmed background config?

**Comment by Andrew42** [comment]
Done

**Comment by kmohrman** [comment]
Ok, thanks @Andrew42. I'd be happy to merge this once the CI finishes.

**Pull request merged**